kntest/pkg/cluster/gke/ops/create.go

```diff
@@ -14,18 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package actions
+package ops
 
 import (
 	"fmt"
 	"log"
 	"strconv"
 
 	container "google.golang.org/api/container/v1beta1"
+	"knative.dev/pkg/test/cmd"
 	"knative.dev/pkg/test/gke"
 	clm "knative.dev/pkg/testutils/clustermanager/e2e-tests"
-	"knative.dev/pkg/testutils/clustermanager/e2e-tests/common"
-	"knative.dev/pkg/testutils/clustermanager/prow-cluster-operation/options"
 	"knative.dev/pkg/testutils/metahelper/client"
```

Comment:
Please consider calling this "metaclient" instead of just client.
Today it's clear enough, but "client" alone is super vague.

Reply:
I think this is the more preferable way to name a Go package, as the upper level package name is `metahelper`, changing the package name to `metaclient` would be duplicate.
One example is https://github.com/knative/eventing/tree/master/pkg/client/injection/client.

Reply:
Sorry, I'm fine with the name of the package.
I mean, in this file, to make "client" less ambiguous, to give it an import alias (or whatever they are called):

```
import (
...
    mhclient "knative.dev/pkg/testutils/metahelper/client"
)
```

I don't really care what the name is, as long as it isn't just "client"; "metahelper" "mhclient" "mh" "mhc"...

Reply:
Changed to `mhclient`